server/bastionai_app/src/main.rs

```diff
@@ -109,7 +185,7 @@ impl RemoteTorch for BastionAIServer {
             tcherror_to_status(artifact.serialize())?
         };
 
-        Ok(stream_data(serialized, 100_000_000).await)
+        Ok(stream_data(serialized, 100_000_000, "Dataset".to_string()).await)
```

Comment:
What is that string for?

Reply:
If you look at the implementation carefully, you can see `stream_data` is used in both `fetch_dataset` and `fetch_model`.
For the logger to log correctly, we have to distinguish between the two.

Reply:
I see and it's better than duplicating the code so I agree with this